lib/modules/manager/flux/readme.md

```diff
@@ -10,7 +10,7 @@ This manager parses [Flux](https://fluxcd.io/) YAML manifests and supports:
 Extracts `helm` dependencies from `HelmRelease` resources.
 
 The `flux` manager extracts `helm` dependencies for `HelmRelease` resources linked to `HelmRepository` or `GitRepository` sources.
-Renovate does support OCI `HelmRepository` sources, those with `type: oci` (in this case extracts `docker` dependencies for referenced `HelmRelease` resources).
+Renovate support OCI `HelmRepository` sources, those with `type: oci`. Renovate will then extract the `docker` dependencies for the referenced `HelmRelease` resources.
```

Comment:
```suggestion
Renovate supports OCI `HelmRepository` sources, those with `type: oci`.
Renovate will then extract the `docker` dependencies for the referenced `HelmRelease` resources.
```

**Changes:**

- Renovate support -> Renovate supports
- One sentence per line

**Background:**

We use the "one sentence per line" pattern in our Markdown files. This makes it easy to move lines around, and to see what changes in new PRs. Please read [Renovate's Style Guide](https://github.com/renovatebot/renovate/blob/main/docs/development/style-guide.md) to learn more.

Reply:
Thanks ! Suggested changes committed.